src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -566,14 +566,14 @@ CharMap =
 #    3gp      3rd Generation Partnership Project            video/3gpp
 #    3g2a     3rd Generation Partnership Project 2 Audio    audio/3gpp2
 #    3g2      3rd Generation Partnership Project 2          video/3gpp2
-#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/aac
+#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/mp4
```

Comment:
@Sami32 I think you miss my point completely. You have license plates on your car because the authorities require you to so that they can easily identify you, but you don't have license plates on your washing machine (I hope) because it isn't a car.